build.sbt

```diff
@@ -149,7 +148,9 @@ lazy val server = project
     Universal / packageBin := (Universal / packageBin)
       .dependsOn(Assets / WebKeys.assets)
       .value,
-    reStart := reStart.dependsOn(Assets / WebKeys.assets).evaluated,
+    Test / test := (Test / test).dependsOn(startElasticsearch).value,
+    reStart := reStart.dependsOn(startElasticsearch).evaluated,
```

Comment:
I think there was an unintended change here - `~server/reStart` in the sbt console no longer constructs assets, so the [dev instructions in the guide for running Scaladex locally](https://github.com/scalacenter/scaladex/blob/main/CONTRIBUTING.md#how-to-run-scaladex-locally) give an unstyled site, with CSS & JS failing to load:

![image](https://user-images.githubusercontent.com/52038/118330234-60e1c100-b4ff-11eb-838a-b0853df1901a.png)

...you might have to run `sbt clean` first to see this problem, if there are still assets lying around from an earlier run.